.kokoro/nightly/common.sh

```diff
@@ -13,6 +13,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# For google-cloud-java, only test specific handwritten libraries included in the monorepo. This is to
+# help speed up the execution as building the entire repo is an expensive operation. Specify the nested
+# `google-cloud-*` path (except for grafeas as it doesn't have one) because maven -pl will only build the
+# specified folder (i.e. parent folder) and ignore all the related sub-modules inside
+google_cloud_java_handwritten_maven_args="java-grafeas,java-vertexai/google-cloud-vertexai,java-resourcemanager/google-cloud-resourcemanager,java-translate/google-cloud-translate"
```

Comment:
This has the handwritten modules written out again. This is separate from the list in the binary script as this also includes the proto modules.